### ISSUE TITLE
evtl. link to ordnance survey mapviewer

### ISSUE THREAD

**Issue by mro** [Megalithicirelandmap/Megalithicirelandmap.github.io#2]
e.g. for [Kilnaruane Carved Pillar Stone](http://megalithicireland.com/Kilnaruane Pillar Stone.html)  Discovery Map 85: V 9842 4753 or http://maps.osi.ie/publicviewer/#V2,498460,547565,11,9

**Comment by mro** [comment]
or [Carrickfergus Castle](http://megalithicireland.com/Carrickfergus%20Castle.html), Discovery Map 15: J 4143 8725 or http://maps.osi.ie/publicviewer/#V2,741339,887187,11,7

**Comment by Megalithicireland** [comment]
Instead of the co-ordinates would you prefer the links to the OSI maps Marcus?

**Comment by mro** [comment]
No, gps is mandatory.

**Comment by Megalithicireland** [comment]
Hi Marcus,
I am just informing you that in the repository for the map it states that
the build is failing. I did quiet a lot of changes to the title pages and
<sym>. I also altered one of two co-ordinates. I don't think the fail is
anything to do with my changes as everything looks great on the map.
No need for a quick response, I know you are probably busy. Just informing
you about the fail message.
Thanks for all your time and effort on Sunday.
Jim
